prow/config/config.yaml

```diff
@@ -2984,6 +2985,7 @@ tide:
         - PingCAP-QE/test-infra
         - PingCAP-QE/test-plan
         - PingCAP-QE/workload
+        - pingcap/tidb-test
```

Comment:
pingcap/tidb-test is not a QE teams's repo, Is this correct?

Reply:
I suggest put it in, similar with test-infra.

Lazy for make big changes. :dog:

Reply:
In the future, It may be in.